springfield/firefox/templates/firefox/more/best-browser.html

```diff
@@ -8,6 +8,8 @@
 
 {% extends "base-protocol.html" %}
 
+{% set params = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=best-browser&utm_content=seo' %}
```

Comment:
```suggestion
{% set params = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=best-browser' %}
```

We should use `utm_content` to tell several links on the same page apart. Since the same value is being applied to all the links on this page we can drop it. (And we don't need to go to the trouble of telling these links apart).